commit 7c70c834cca22ad6ce1f6b33ccdd507f31fbfdd2
Author: Silvio Rhatto <rhatto@riseup.net>
Date:   Mon Feb 27 17:17:56 2012 -0300

    Force rsync handler run bash at the remote destination (upstream #3003)

diff --git a/handlers/rsync.in b/handlers/rsync.in
index b2c4b59..4905ace 100644
--- a/handlers/rsync.in
+++ b/handlers/rsync.in
@@ -282,8 +282,9 @@ function eval_config {
      str="${i//__star__/*}"
      excludes="${excludes} --exclude='$str'"
   done
-  IFS=$SAVEIFS
-  set +o noglob
+
+  # Make sure we'll run bash at the destination
+  ssh_cmd="$ssh_cmd /bin/bash"
 }
 
 function rotate_short {
